uui/components/navigation/MainMenu/Burger/BurgerButton.tsx

```diff
@@ -12,7 +12,7 @@ export const BurgerButton = withMods<ButtonProps, BurgerButtonMods>(
     Button,
     (props) => [
         css.root,
-        'uui-mainMenu-burger-button',
```

Comment:
let's avoid this change to not cause a breaking change

Reply:
this is not a breaking change, it is implemented within the framework of this RC, I just tweaked the class, it's more correct than using a camelcase